packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -129,6 +129,10 @@ export default function () {
   return {
     inherits: require("babel-plugin-transform-strict-mode"),
 
+    manipulateOptions(opts, parserOpts) {
+      parserOpts.plugins.push("objectRestSpread");
+    },
```

Comment:
Yes, just added Object.assign with `babel-plugin-transform-object-assign` as devDep to support node < 4.